hicommonwealth/commonwealth
# Fixed CI linter to run in parallel

## Link to Issue
Closes: #8945

## Test Plan
- Make a type error, check that `pnpm -r check-all` catches the type error and throws an error.
- Create a build error (for example type some non-ts keyword in a file). Make sure that `pnpm -r check-all` catches this as well.
